releasenotes/notes/Update-Java-crash-and-OOME-integration-001a00c49ad3bf80.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This change is about the admission controller of the cluster-agent so this should be in the release notes of the cluster-agent, not the agent.